src/lib/analytics/analytics.js

```diff
@@ -183,6 +183,7 @@ const patchLogger = () => {
     }
     if (Config.sentryDSN && Config.env !== 'test') {
       reportToSentry(logMessage, {
+        logMessage,
```

Comment:
1. this seems like a duplicate
2.   in reporttosentry we have this:
Sentry.captureException(new Error(errorMsg))
this seems wrong, if we have an errorobj we should not create a new one

Reply:
@sirpy 

1) will remove it

Reply:
2. if errorObj exists and is of type error use it, otherwise create it